lib/course_planner_web/views/shared_view.ex

```diff
@@ -283,13 +283,15 @@ defmodule CoursePlannerWeb.SharedView do
 
   # show pages
 
-  def user_list(users, opts \\ []) do
+  def user_list(current_user, users, opts \\ []) do
+    clickable = current_user.role == "Coordinator"
     empty_text = opts[:empty_text] || "There are no users here yet"
     render "user_list.html", users: users,
-                             empty_text: empty_text
+                             empty_text: empty_text,
+                             clickable: clickable
   end
 
-  def user_bubble(user) do
+  def user_bubble(user, clickable) do
```

Comment:
what if we don't change the `lib/course_planner_web/templates/shared/user_bubble.html.eex` template, and we change the url instead to blank if it is not clickable?

Reply:
so you mean i pass a empty url to the user_bubble.html

Reply:
done